src/pages/TrueSight/components/FilterBar/TrueSightSearchBox.tsx

```diff
@@ -13,6 +13,7 @@ import { TrueSightTokenData } from 'pages/TrueSight/hooks/useGetTrendingSoonData
 import { TruncatedText } from 'pages/TrueSight/components/TrendingSoonLayout/TrendingSoonTokenItem'
 
 interface TrueSightSearchBoxProps {
+  className?: string
```

Comment:
Why use className here?

Reply:
Use it so it can be extended with styled-components